src/awakening_stage/gui/InventoryScreen.tscn

```diff
@@ -293,12 +293,12 @@ text = "CRAFTING_TAKE_ALL"
 anims/Flash = SubResource( 1 )
 anims/RESET = SubResource( 2 )
 
-[connection signal="Closed" from="GroundPanel" to="." method="OnGroundPanelClosed"]
-[connection signal="Closed" from="MainInventory" to="." method="OnInventoryPanelClosed"]
+[connection signal="Dismissed" from="GroundPanel" to="." method="OnGroundPanelClosed"]
```

Comment:
What's the reason why the this signal was renamed? I think "closed" is pretty simple and very clear for when a popup emits that signal.

Reply:
Due to ambiguity with the new `CustomPopup.Close` and for reinforcing what I had documented in this signal's xml summary.

Reply:
Okay, I read the documentation there. But now I feel like in 100% of cases I've seen the X closing the dialog needs to be handled the same way as pressing the cancel button or hitting ESC. So having a separate dismiss operation that is called a bunch doesn't seems very useful. Or did you mean it as a replacement for canceling? If that's the case I think the name should be like "Canceled" for clarity.

Reply:
Keep in mind this class is one level below confirmation dialogs, so it shouldn't know what "canceling" is.